### DISCUSSION TITLE
Hidden files sync and plugin sync

### DISCUSSION THREAD

**Discussion by MohamedBassem** [vrtmrz/obsidian-livesync#95] [Q&A]
Hi, I noticed that recently you released a feature for synchronizing hidden files in the vault. Those hidden files include stuff like the installed plugins, core plugin settings and general vault settings. At this point, is there still a point in having both plugins and hidden files sync enabled at the same time? It seems that the hidden files sync is much more generic and should be enough?

**Comment by vrtmrz** [comment]
Yes, You're right. But hidden file sync is too generic now, and I would love to merge features in a while. 
We miss these features now: 
- Keep older versions of plugins and/or their settings. 
I often change the device and vault name to save circumstances. 
This would be solved as the feature: Hidden files history. 
- Apply only main or data. 
This could be useful between mobile and desktop synchronisation. 
This is half implemented now, but I'll make it easier.

**Comment by hilsonp** [comment]
As a new user, I didn't know that the "sync plugin" feature was older than the "sync hidden files" feature.
At first, I enabled the "sync plugin" one but would not understand clearly the UI (having 2 laptop and one iphone).

After some reading, I disabled "plugin sync" and enabled the "hidden files sync".

Why aren't the css snippets synced ? Aren't they regular files in an hidden folder ?

**Reply by vrtmrz** [reply]
Thank you for asking me!
CSS snippets are able to be synchronised using "Sync hidden files". 
Have your CSS snippets been synchronised at least once and are now being ignored now? or, they have not been synchronised at all?

**Comment by hilsonp** [comment]
Hello. 
Thank you for your fast reply.
My CSS snippets did not synchronize at all.
I'll retry a bit and will open an issue to ease the follow-up.
Best regards,

**Reply by dliberalesso** [reply]
Test it with version 0.17.15. It seems there was a bug syncing hidden files that was fixed.
